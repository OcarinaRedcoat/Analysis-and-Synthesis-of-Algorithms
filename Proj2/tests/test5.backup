id: 2 -- capacity: 3
id: 3 -- capacity: 15
id: 4 -- capacity: 5
id: 5 -- capacity: 4
id: 6 -- capacity: 0
id: 7 -- capacity: 0
orin: 2 -- dest: 7 -- capacity: 0
orin: 2 -- dest: 6 -- capacity: 0
orin: 3 -- dest: 7 -- capacity: 0
orin: 3 -- dest: 6 -- capacity: 0
orin: 3 -- dest: 5 -- capacity: 0
orin: 4 -- dest: 6 -- capacity: 0
orin: 4 -- dest: 5 -- capacity: 4
orin: 5 -- dest: 1 -- capacity: 5
orin: 6 -- dest: 1 -- capacity: 2
orin: 7 -- dest: 1 -- capacity: 2