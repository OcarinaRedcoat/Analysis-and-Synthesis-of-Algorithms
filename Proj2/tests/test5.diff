2,3c2
< 7
< 2 6
---
> 67
5d3
< 3 6
