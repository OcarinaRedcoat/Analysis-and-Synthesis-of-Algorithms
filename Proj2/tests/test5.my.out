12
67
3 5
